physionet-django/console/templates/console/event_agreement_new_version.html

```diff
@@ -0,0 +1,23 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}{{ SITE_NAME }} - Event Agreements{% endblock %}
+
+{% block content %}
+  <div class="card mb-3">
+    <div class="card-header">
+        {{ event_agreement.name }} - new version
+    </div>
+    <div class="card-body">
+        <form method="post" action="{% url 'event_agreement_new_version' event_agreement.pk %}">
+            {% include "inline_form_snippet.html" with form=event_agreement_form %}
```

Comment:
Right forgot this - you need to change this to `content_inline_form_snippet.html` for the help text to display on this page

Reply:
Oh yes, i missed this, and just did this on the listing page. 
Fixing this right now